pages/facts.html

```diff
@@ -1,12 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Facts About Dolly</title>
+  <link rel="icon" href="../images/butterflysmoky-xxl.png" type="image/x-icon">
+  <link href="../styles/index.css" rel="stylesheet">
+  <link href="../styles/facts.css" rel="stylesheet">
 </head>
+
 <body>
+  <div class="sidenav">
+    <a href="../index.html">Home</a>
+    <a href="gallery.html">Gallery</a>
+    <a href="philanthropy.html">Philanthropy</a>
+    <a href="discography.html">Discography</a>
+  </div>
   
+  <div class="dropdown">
+    <strong>
```

Comment:
Attempting to validate this file, there's an error that `<strong>` can't have nested elements. It's okay to use the `<strong>` tag if it's semantic to do so, but be sure to follow the HTML5 spec!